### ISSUE TITLE
Idea for next version

### ISSUE THREAD

**Issue by mikebolt** [molarmanful/gCube#15]
I just finished getting NxNxN cubes working with ThreeTwist. After cleaning up the code a bit I would like to work on integrating it into gCube.

I was looking through the thread on speedsolving.net and I eventually found [several other JavaScript cube "applets"](https://github.com/blonkm/rubiks-cube#overview-of-online-cubes). I think it would be cool to support all of these applets. gCube could be a common interface for these applets.

I really like the idea of using custom HTML5 elements to configure the cube, then simply including the .js file or perhaps calling a single function to display the cubes, and I think that's gCube's greatest strength. One of the attributes of the g-cube tag could be something like "renderer" or "library", so you could choose between e.g. GLube, Twisty.js, and Cuber.

This would be a major change, so it should be a part of the "next version" of gCube.

**Comment by molarmanful** [comment]
Yeah, this will be good. When you finish integration, I'll create some Codepens to test your work.

Thanks!
